lib/internal/test_runner/mock.js

```diff
@@ -202,6 +202,60 @@ class MockTracker {
     return mock;
   }
 
+  getter(
+    object,
+    methodName,
+    implementation = kDefaultFunction,
+    options = kEmptyObject
+  ) {
+    if (implementation !== null && typeof implementation === 'object') {
+      options = implementation;
+      implementation = kDefaultFunction;
+    }
+
+    const { getter = true } = options;
+
+    validateBoolean(getter, 'options.getter');
+
+    if (!getter) {
```

Comment:
We can simplify this and let `method()` handle the validation (same for setter).

```suggestion
    if (options?.getter === false) {
```

Reply:
I didn't understand the reason/logic behind allowing only `getter === true` while setting it to true as default, but if it's `false` throw an error? Why do we even check for `options?.getter` at all?

Reply:
It is incorrect to specify `options.getter` for `getter()`. If we don't check it, `getter()` will not throw an error even if we specify `false` for `options.getter`.  I think that it is better to check `options.getter`, since getter() will not throw an error if `options.getter` is specified as `false`.